pom.xml

```diff
@@ -63,6 +63,7 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <github.global.server>github</github.global.server>
     <site.installationModule>google-cloud-game-servers-v1beta-parent</site.installationModule>
+    <google.core.version>1.92.5</google.core.version>
```

Comment:
I don't think we need to manage the version here as we are using the [google-cloud-shared-dependencies-bom](https://github.com/googleapis/java-game-servers/blob/35e8ffcae005549887f55d771196939508e82ab6/pom.xml#L99) which provides a version for `core`

Reply:
done